### DISCUSSION TITLE
What will happen when the task is still evaluating at the end time of phase3?

### DISCUSSION THREAD

**Discussion by hbezlls** [PaddlePaddle/PaddleSpatial#290] [BaiduKDDCup2022] [answered]
Dear organizer,

The submit page has been changed after July 18 23:59 AOE, but we still have one task evaluating(Pending Timeout once before).  Will the score changed later?

**Comment by ramber1836** [comment, marked as answer]
Yes, your submitted job will continue being evaluated if everything goes well.

**Reply by hbezlls** [reply]
> 

Thanks for your reply. Where can I find the submit result when the job finished?

**Reply by ramber1836** [reply]
We leave the leaderboard open till tomorrow night so that you can find out whether your last submission goes well.
